src/controllers/email-controller.js

```diff
@@ -0,0 +1,58 @@
+const Accounts = require('../models/accounts');
+const {StatusCodes} = require ('http-status-codes');
+const {generateCode} = require('../helpers/generateCode');
+const nodemailer = require('nodemailer');
+const {responseServer, responseGeneral} = require('../helpers/response-result');
+
+const postEmail = async (request, response) => {
+    const {email} = request.body;
+    let codeConfirmation = generateCode();
+    Accounts.findOne({email:email}, {_id:1, name:1, lastname:1})
+    .then(async (account) =>{  
+        if(account){
+            await Accounts.updateOne({_id:account._id}, {codeConfirmation:codeConfirmation});
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
            await Accounts.updateOne({_id:account._id}, {codeConfirmation});
```